docs/source/loading_datasets.rst

```diff
@@ -218,6 +250,13 @@ Here is an example loading two CSV file to create a ``train`` split (default spl
     >>> from datasets import load_dataset
     >>> dataset = load_dataset('csv', data_files=['my_file_1.csv', 'my_file_2.csv'])
 
+You can also provide the URLs of remote csv files:
+
+.. code-block::
+
+    >>> from datasets import load_dataset
+    >>> dataset = load_dataset('csv', data_files="https://huggingface.co/datasets/lhoestq/demo1/resolve/main/data/train.csv")
```

Comment:
i'm not sure where we should mention this, but showing that you can download from private repos with `use_auth_token=True` would be useful

Reply:
I added a note at https://github.com/huggingface/datasets/blob/01f53281b267a1020bbb3b2ecaa0848f376897e8/docs/source/loading_datasets.rst#from-local-or-remote-files